README.md

```diff
@@ -74,10 +74,32 @@ e.g. the indication of a lifecycle state of a value by “as-specified” or “
 e.g. the indication of multiplicity of allowed components for modelling by “greater than 0” and others.
 (Answered: 2020-08-13)
 
-**Where are examples of AAS available?**
+**Where are examples of AAS available?** <!-- ID: 16 -->
 
 Examples of several suppliers are found at: http://admin-shell-io.com/samples/ 
 (Answered: 2020-08-13)
 
+**What are best practices for creating custom IRI identifiers for generic concepts.** <a name="id18"></a><!-- ID: 18 -->
```

Comment:
2020-08-24 Meeting Exemplarische Modellierung: check whether <a name=...> still valid, seems to be obsolete.